templates/game.eex

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>Elixir TicTacToe</title>
+    <style>
+
+body {
+  background-color: black;
+  color: white;
+}
+
+h1 {
+  text-align: center;
+}
+
+#board {
+  background-color: #4d4d4d;
```

Comment:
No reason, I decided to postpone separating styling into a static file and polishing the user-interface to later on, though it makes sense to do it now, which I shall.

Thank you! :)